### PR TITLE
Drop nvtx test

### PR DESCRIPTION
**Pull request by carmocca**
## What does this PR do?

All of our runners have higher device capabilities so this test making it effectively untested. This PR removes it

Proposed by @Borda in #19141

----
📚 Documentation preview 📚: https://pytorch-lightning--19154.org.readthedocs.build/en/19154/

cc @carmocca @borda

### CODE REVIEW

File: tests/run_standalone_tests.sh [hunk 1]
```diff
@@ -69,7 +69,7 @@ for i in "${!parametrizations_arr[@]}"; do
   # execute the test in the background
   # redirect to a log file that buffers test output. since the tests will run in the background, we cannot let them
   # output to std{out,err} because the outputs would be garbled together
-  python ${defaults} "$parametrization" &>> standalone_test_output.txt &
```
Comment (carmocca) [inline tests/run_standalone_tests.sh:72]: https://github.com/Lightning-AI/pytorch-lightning/pull/19141/files breaks TPU standalone tests. The TPU machines do not include the `python` to `python3` mapping

Comment (Borda) [reply tests/run_standalone_tests.sh:72]: if we do not keep breaking TPU tests we would see it comming

File: tests/tests_pytorch/profilers/test_profiler.py [hunk 2]
```diff
@@ -471,24 +471,6 @@ def look_for_trace(trace_dir):
     assert look_for_trace(tmpdir / "lightning_logs" / "version_0")
 
 
-# Todo: this test has not been running as all our CI GPU runners have higher capacity
```
Comment (Borda) [inline tests/tests_pytorch/profilers/test_profiler.py:474]: So we would never take it back?

Comment (carmocca) [reply tests/tests_pytorch/profilers/test_profiler.py:474]: No. The test is not great anyways. Eventually device capability 8 will be ancient

**Review by awaelchli — APPROVED** [review]

**Review by Borda — APPROVED** [review]

**Pull request merged**